commands/util/binary.js

```diff
@@ -0,0 +1,24 @@
+const { SlashCommandBuilder, EmbedBuilder } = require("discord.js");
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("binary")
+    .setDescription("Convert numbers into binary")
+    .addIntegerOption(option =>
+        option
+            .setName('integer')
```

Comment:
Nit: Could you rename the option to "number" or "num"? Something that's a bit more friendly than integer